SkietbaanBE/SkietbaanBE/Controllers/CompetitionController.cs

```diff
@@ -32,8 +31,16 @@ public IEnumerable<Competition> Get()
             // return _context.Competitions.ToArray<Competition>();
```

Comment:
I know this is not your code but please remove GC

Reply:
Thank you, will do.